src/app/shared/enum/enumUA/admins.ts

```diff
@@ -0,0 +1,18 @@
+export enum AdminRoleUkr {
+    ministryAdmin = 'Адміністратор міністерства',
+    regionAdmin = 'Адміністратор регіона',
+    territorialCommunityAdmin = 'Адміністратор ОТГ'
+
+  }
+
+export enum AdminRoleUkrReverse {
+  'Адміністратор міністерства' = 'ministryAdmin',
+  'Адміністратор регіона' = 'regionAdmin',
+  'Адміністратор ОТГ' = 'territorialCommunityAdmin'
+}
+
+export enum AdminTitles {
+    NeverLogged = 'Очікує логування',
```

Comment:
I believe there is too much space, it should be 2 spaces

Reply:
fixed